### PR TITLE
Factions may now have their own specific job blacklists for species + some blacklists

### PR DESCRIPTION
**Pull request by mikomyazaki**
Resubmitting #8334 which was reverted, with corrections.

Need to get someone to check if the blacklists I took from that PR are still correct.

Issue with previous PR was that it was resetting player jobs if ANY job was banned from that species. And so since all xenos are banned from Captain, when the player would try to be NanoTransen it would reset their jobs.

### CODE REVIEW

**Comment by mikomyazaki** [comment]
!review

File: html/changelogs/paradoxspace-t1000.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+author: ParadoxSpace & mikomyazaki
```
Comment (NonQueueingMatt) [inline html/changelogs/paradoxspace-t1000.yml:1]: is this even snakebitten's work at this point

Comment (mikomyazaki) [reply html/changelogs/paradoxspace-t1000.yml:1]: Yes, slightly. He did contribute and this PR would not have been made without him.

**Review by fernerr — APPROVED** [review]

**Comment by mikomyazaki** [comment]
DNM I need to add more limits from a thing Matt gave me

**Comment by mikomyazaki** [comment]
!review

**Pull request merged**